Chinese/DesertHell/OldStory/Cat2Dog/变成狗了.md

```diff
@@ -1,2 +1,2 @@
-这只猫变成了一只狗。它很茫然。它就去找白胡子老头，白胡子老头说，说什么呢？
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.